doc/dev/BUILD_WASM.md

```diff
@@ -4,7 +4,7 @@ F3D can be built in WebAssembly using emscripten in order to embed it into a web
 It is still experimental and only a small fraction of the libf3d public API is bound.
 An example can be seen at https://f3d.app/web
 
-This guide is describing how to build VTK and F3D with emscripten using Docker on Linux.
+This guide is describing how to build VTK and F3D with emscripten using Docker on Linux or Windows (WSL).
```

Comment:
WSL is not Windows, not sure its worth mentionning.

Reply:
It's using the WSL layer silently through Docker. I think it's worth mentioning this guide is valid to run on Windows.

Reply:
I can remove ` (WSL)` if you think it's better tho

Reply:
Do the command need to be run in a WSL terminal or does it work in any terminal assuming docker is installed and in the PATH ?

Reply:
Any terminal. If WSL is not installed, Docker will refuse to run the image.